pjlib/src/pj/ioqueue_winnt.c

```diff
@@ -133,12 +133,9 @@ struct pj_ioqueue_key_t
     int                 connecting;
 #endif
 
-#if PJ_IOQUEUE_HAS_SAFE_UNREG
     pj_atomic_t        *ref_count;
```

Comment:
I don't think we should remove all the code related to `PJ_IOQUEUE_HAS_SAFE_UNREG`. I believe what @nanangizz meant was that we need to make sure `PJ_IOQUEUE_HAS_SAFE_UNREG` is set to 1.

Since by default PJ_IOQUEUE_HAS_SAFE_UNREG is already 1, IMO adding a check below should be sufficient, just to make sure that users don't accidentally disable it:
#if PJ_IOQUEUE_HAS_SAFE_UNREG == 0
    /* IOCP only works with PJ_IOQUEUE_HAS_SAFE_UNREG enabled, otherwise we will get memory error in ... */
    #error "PJ_IOQUEUE_HAS_SAFE_UNREG must be enabled to use ioqueue IOCP"
#endif

Another alternative is to force enable it in `pjlib/config.h`, something like:
```
#if PJ_IOQUEUE_IMP == PJ_IOQUEUE_IOCP
#   define PJ_IOQUEUE_HAS_SAFE_UNREG    1
#endif
```
But this is more complicated since we need to define `PJ_IOQUEUE_IMP` similar to `PJ_SSL_SOCK_IMP`.

Reply:
It's still there in all other ioq backends.